cds_ils/importer/XMLRecordToJson.py

```diff
@@ -54,14 +54,17 @@ def dump(self):
             is_deletable = False
 
         init_fields = {}
-        if "im" in marc_record.get("leader", []):
+        leader_tag = marc_record.get("leader", [])
+        if "am" in leader_tag:
```

Comment:
This was added to be able to fail with an error if some unknown media type is provided, but to still handle `am` as an `e-book`